exporter/awss3exporter/config.go

```diff
@@ -63,6 +66,9 @@ type S3UploaderConfig struct {
 	// If unspecified, a default function will be used that generates a random string.
 	// Valid values are: "uuidv7"
 	UniqueKeyFuncName string `mapstructure:"unique_key_func_name"`
+
+	// Not in config file, will be set in [Config.Validate].
```

Comment:
typically Validate is idempotent. Do you need this field on the Config struct? Can it be derived when used?

Reply:
I can change that, good point

Reply:
Moved the logic to check timezone into `newUploadManager`